media-plugins/gst-plugins-qt5/gst-plugins-qt5-1.18.4.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+GST_ORG_MODULE=gst-plugins-good
+
+inherit gstreamer-meson
+
+DESCRIPTION="Qt5 QML video sink plugin for GStreamer"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="+egl wayland +X orc"
+
+REQUIRED_USE="
+	wayland? ( egl )
+"
+
+RDEPEND="
+	>=media-libs/gst-plugins-base-${PV}:${SLOT}[egl?,opengl,wayland?,X?]
+	media-libs/mesa[egl?,X?,${MULTILIB_USEDEP}]
```

Comment:
`${MULTILIB_USEDEP}` doesn't make sense if this can't be multilib anyway due to Qt5.